atlas-content-modeler/config.json

```diff
@@ -0,0 +1,107 @@
+{
+  "title": "Atlas Content Modeler",
+  "description": "Create content models and create taxonomies and relationships.",
```

Comment:
This description should be similar in length to the other products. Shoot for around 30-45 words.